apps/ffdweb-site/src/app/_styles/globals.css

```diff
@@ -28,6 +28,16 @@
   --color-brand-secondary-800: #002d00;
   --color-brand-secondary-900: #001100;
 
+  --color-brand-neutral-white: #ffffff;
+  --color-brand-neutral-100: #eaeaea;
+  --color-brand-neutral-200: #c6c6c6;
+  --color-brand-neutral-300: #a3a3a3;
+  --color-brand-neutral-400: #818181;
+  --color-brand-neutral-500: #616161;
+  --color-brand-neutral-600: #434343;
+  --color-brand-neutral-700: #262626;
+  --color-brand-neutral-black: #0d0d0d;
```

Comment:
No need to add these - We are using Tailwind's built-in `neutral` at the moment.